openedx/features/course_card/helpers.py

```diff
@@ -101,18 +105,20 @@ def get_course_cards_list():
     return courses_list
 
 
-def is_course_in_programs(course_id):
+def is_course_in_programs(parent_course_key):
     """
-    Helper function to check if course is part of program
-    @param course_id: course key
-    @return: true if course is part of program otherwise false
+    Helper function to check if course is part of program.
+    @param parent_course_key: parent course key
+    @return: True if parent of any course reruns from discovery matches
```

Comment:
Actually we are checking just first re-run so this line is misleading. How about `@return: True if parent of first course reruns from discovery matches` and then in description in the very first line of doc string we can describe why are we checking just first rerun like `Parent course(course card) of each rerun is same so we just compare with parent of first rerun and check if course is part of the program.`

Reply:
'any' covers the case of first course rerurns as well. I think mentioning to check first rerun seems to go against the logic. thoughts?